cjio/cityjson.py

```diff
@@ -222,7 +222,7 @@ def __init__(self, file=None, j=None, ignore_duplicate_keys=False):
             self.j = j
             self.cityobjects = {}
             self.path = None
-            self.reference_date = datetime.now()
+            self.reference_date = datetime.now().strftime('%Y-%m-%d')
```

Comment:
This should be left as it was before, as `generate_metadata` expects a `datetime` argument.

Reply:
That would be inconsistent with what was in https://github.com/cityjson/cjio/blob/65e5a85f69d333be6deb882565bc543b7cc545f7/cjio/cityjson.py#L219 so better make a string and change `generate_metadata` to expect a string.

Also, because we cannot serialize `datetime` to json so better keep them as strings to avoid errors.

Reply:
> That would be inconsistent with what was in
> 
> https://github.com/cityjson/cjio/blob/65e5a85f69d333be6deb882565bc543b7cc545f7/cjio/cityjson.py#L219
> so better make a string and change `generate_metadata` to expect a string.

But in that case we take care of `datetime` formatting in two different places: one in Line 219 of `cityjson.py` and one in `generate_metadata`. Since `generate_metadata` will always have to format a `datetime` to an `str` anyway (in case a filename is provided) I suggest that it takes care of formatting the `reference_date` as well. What do you think?

> Also, because we cannot serialize `datetime` to json so better keep them as strings to avoid errors.

As far as I understand, `reference_date` is not really going to be serialized anywhere, is it? I think it's just a variable of the `CityJSON` class which is not written in the file.